File: src/main/java/net/shoal/sir/voteup/task/VoteEndTask.java
package net.shoal.sir.voteup.task;

import net.shoal.sir.voteup.config.VoteManager;
import org.bukkit.scheduler.BukkitRunnable;

public class VoteEndTask extends BukkitRunnable {

    private String voteID;
    public VoteEndTask(String id) {
        this.voteID = id;
    }

    @Override
    public void run() {
        VoteManager.getInstance().voteEnd(voteID);
    }
}

File: VoteUp/src/main/java/net/shoal/sir/voteup/task/VoteEndTask.kt
package net.shoal.sir.voteup.task

import net.shoal.sir.voteup.api.VoteUpAPI
import org.bukkit.scheduler.BukkitRunnable

class VoteEndTask(private val voteID: String?) : BukkitRunnable() {
    override fun run() {
        VoteUpAPI.VOTE_MANAGER!!.endVote(voteID)
    }

}